numpy/core/tests/test_arrayprint.py

```diff
@@ -399,6 +403,15 @@ def test_wide_element(self):
             "[ 'xxxxx']"
         )
 
+    @hypothesis.given(hypothesis.extra.numpy.from_dtype(np.dtype("U")))
+    def test_any_text(self, text):
```

Comment:
My main question here is: are the `text` strings that are generated by `hypothesis` 100% reproducible over time/versions? And if not, what's the workflow to deal with a bug report for some random string by random user on some OS/install method/etc.?

Reply:
Thanks. Commit https://github.com/numpy/numpy/pull/14440/commits/28887ac29f8764144e3c51d4ddd4a7ac1c77277f should address this.

I tested the reproducibility with the following approach:
Before making the change in https://github.com/numpy/numpy/pull/14440/commits/28887ac29f8764144e3c51d4ddd4a7ac1c77277f
1. Add a side-effect in the test to append the `text` to a file `~/text.txt`
2. Set `PYTHONHASHSEED=random`
3. Run the test once, `mv ~/text.txt ~/text0.txt`
4. Run the test again, `diff ~/text.txt ~/text0.txt` -> The files are different.
5. Then make the change in https://github.com/numpy/numpy/pull/14440/commits/28887ac29f8764144e3c51d4ddd4a7ac1c77277f
6. Perform step 3 and 4, the files are now identical.

Reply:
We can also use the [`@example`](https://hypothesis.readthedocs.io/en/latest/reproducing.html#providing-explicit-examples) directive to force a known failing case into the test once reported by a user or by the CI system